config.md

```diff
@@ -340,6 +340,17 @@ For Linux-based systems, the `process` object supports the following process-spe
 
     * **`class`** (string, REQUIRED) specifies the I/O scheduling class. Possible values are `IOPRIO_CLASS_RT`, `IOPRIO_CLASS_BE`, and `IOPRIO_CLASS_IDLE`.
     * **`priority`** (int, REQUIRED) specifies the priority level within the class. The value should be an integer ranging from 0 (highest) to 7 (lowest).
+* **`cpuAffinity`** (object, OPTIONAL) specifies CPU affinity used to execute the process.
+    The following properties are available:
+    * **`initial`** (string, REQUIRED) is a list of CPUs a runtime parent
+      process to be run on initially, before the transition to container's
+      cgroup. This is a a comma-separated list, with dashes to represent
+      ranges. For example, `0-3,7` represents CPUs 0,1,2,3, and 7.
+    * **`final`** (string, OPTIONAL) is a list of CPUs the process will be run
+      on after the transition to container's cgroup. The format is either the
+      same as for `initial`, or a special value `cgroup` which means container's
+      CPU affinity, as defined by [cpu.cpus property](./config.md#configLinuxCPUs)).
+      If omitted or empty, the `initial` setting is kept.
```

Comment:
shouldn't it be the opposite? I mean "if ommitted, cpu.cpus will be in effect"?
The `initial` seems to allow to set affinity to cpu cores that are outside of the set which is specified by `cpu.cpus`.
This also trigger the question of validation of the values between `cpuAffinity` and `cpu.cpus`.

Reply:
There are two questions here.

1. What is the default if `final` is not set. You are probably right that `cpu.cpus` makes more sense. UPDATED.

2. Whether to allow affinity that's outside of container's cpu.cpus. While it seems theoretically possible for the `initial` cpuset, I haven't checked that it works, and I don't want to add any specific constraints to the spec (or to runc/crun).

Reply:
PTAL @kad